runtime/isp_run_app.py

```diff
@@ -236,7 +237,7 @@ def main():
 
     pex_path = args.pex
     if not pex_path:
-        pex_path = os.path.join(run_dir, "pex-kernel")
+        pex_path = os.path.join(run_dir, os.path.basename(sim_module.defaultPexPath(policy_name, arch, args.extra)))
```

Comment:
What if the user send a bunch of random chars as part of the args.extra, would the pex path be found?

Reply:
That's the directory where the PEX kernel is to be compiled, so I imagine so.